docs/source/releases/v1_12_0a0.rst

```diff
@@ -0,0 +1,33 @@
+ | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+ | # # #
+ | # # # Author:
+ | # # # Naval Research Laboratory, Marine Meteorology Division
+ | # # #
+ | # # # This program is free software: you can redistribute it and/or modify it under
+ | # # # the terms of the NRLMMD License included with this program. This program is
+ | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+ | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+ | # # # for more details. If you did not receive the license, for more information see:
+ | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Version 1.12.0a0 (2024-02-12)
+*****************************
+
+* Refactor: Clean up optional dependencies throughout the codebase.
+
+Refactoring Updates
+-------------------
+
+Clean up optional dependencies throughout the code base
```

Comment:
Don't merge until this [PR](https://github.com/NRLMMD-GEOIPS/geoips/pull/438) has been merged.

Reply:
Ready now.